css/style.css

```diff
@@ -1071,3 +1356,1390 @@ ul {
+    background-image: url("images/desktop.jpg");
+    background-size: cover;
+    background-position: center;
+  }
+
+  #app-bar-compact {
+    width: 100%;
+    aspect-ratio: 1440 / 72;
+    z-index: 3;
+  }
+
+  #div-logo {
+    width: 6.944444%;
+    aspect-ratio: 100 / 40;
+    left: 9.930556%;
+    top: 0.292683%;
+    position: absolute;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    font-size: 100%;
+  }
+
+  #div-logo-a {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #6070ff;
+    font-weight: 700;
+    font-size: 1.25vw;
+    line-height: 112%;
+  }
+
+  #icon-button-button:disabled {
+    background: transparent;
+  }
+
+  #div-button {
+    width: 0;
+    left: 0%;
+    top: 0%;
+    border: none;
+  }
+
+  #menu {
+    width: 0%;
+    aspect-ratio: 1 / 1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .vector {
+    width: 0;
+    border: none;
+    background: transparent;
+  }
+
+  #menu-2 {
+    position: absolute;
+    width: 17.986111%;
+    aspect-ratio: 259 / 40;
+    left: 72%;
+    top: 0.312195%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    gap: 1.4vw;
+  }
+
+  #menu-2-button {
+    background: transparent;
+    border: none;
+    color: #344563;
+    font-weight: 600;
+    font-size: 1.2vw;
+    line-height: 100%;
+  }
+
+  #headline {
+    position: absolute;
+    width: 100%;
+    aspect-ratio: 1440 / 846;
+    left: 0%;
+    top: 1.385366%;
+    z-index: 2;
+    margin: 0;
+    border-bottom-left-radius: 10vw;
+  }
+
+  #div-primary-text-1 {
+    position: absolute;
+    width: 33%;
+    aspect-ratio: 920 / 122;
+    left: 18.055556%;
+    top: 36%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: normal;
+  }
+
+  #div-primary-text-2 {
+    position: absolute;
+    width: 63.888889%;
+    aspect-ratio: 920 / 98;
+    left: 18.055556%;
+    top: 48.7%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .primary-text-1 span {
+    display: none;
+  }
+
+  .primary-text-1::after {
+    position: absolute;
+    top: -20%;
+    height: 100%;
+    width: 100%;
+    font-weight: 700;
+    font-size: 3.4vw;
+    line-height: 4vw;
+    color: #172b4d;
+    align-items: center;
+    content: "I'm Alexandra Glad to see you!";
+  }
+
+  .primary-text-2 {
+    height: 100%;
+    width: 100%;
+    font-weight: 400;
+    font-size: 1.38vw;
+    line-height: 2vw;
+    text-align: left;
+    color: #344563;
+  }
+
+  #div-social-media {
+    position: absolute;
+    width: 8.125%;
+    aspect-ratio: 117 / 24;
+    left: 18.055556%;
+    top: 63%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  }
```

Comment:
Kindly adjust the display of the social icons in the hero section to correspond to the figma design template. `Let's Connect` overlaps with the social media icons.

Also, kindly make the text colors to match.

Your design:
![image](https://user-images.githubusercontent.com/20554583/194474514-c9f74bfc-f6f7-4788-a535-508ffecdf01b.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/194474551-a07a943b-8b2f-488c-81fe-c93f963a3bd0.png)